packages/@vue/eslint-config-prettier/index.js

```diff
@@ -6,5 +6,9 @@ module.exports = {
   ],
   rules: {
     'prettier/prettier': 'warn'
+  },
+  globals: {
```

Comment:
I think you should add `commonjs` to ESLint's `env` options or add a `.eslintignore` file instead of adding globals manually.